ivadomed/scripts/prepare_dataset_vertebral_labeling.py

```diff
@@ -62,21 +62,20 @@ def extract_mid_slice_and_convert_coordinates_to_heatmaps(path, suffix, aim=-1):
     Returns:
         None. Images are saved in BIDS folder
     """
-    t = os.listdir(path)
-    t.remove('derivatives')
+    t = [d.name for d in Path(path).iterdir() if d.name != 'derivatives']
```

Comment:
Could you exapnd `t`, `d` and also `i` below? Whenever we see single letter variable we should start correcting them for clarifying and eliminate this anti-pattern.